include/asterisk/res_pjsip.h

```diff
@@ -3500,6 +3514,53 @@ struct ast_threadpool *ast_sip_threadpool(void);
  */
 struct ast_sip_transport_state *ast_sip_get_transport_state(const char *transport_id);
 
+/*!
+ * \brief Return the SIP URI of the Contact header
+ * 
+ * @param tdata
```

Comment:
Why the use of @param versus our normal \param

Reply:
Sorry about that. Done.